apps/paymaster-proxy/src/errors/JsonRpcCastableError.ts

```diff
@@ -0,0 +1,9 @@
+export abstract class JsonRpcCastableError extends Error {
```

Comment:
all errors handled during business logic will throw this a child of this error so that it can return the correct code / error message